### DISCUSSION TITLE
May I ask the contact email of the main person in charge or manager of nvm-window?

### DISCUSSION THREAD

**Discussion by yt46767** [coreybutler/nvm-windows#1004] [Support] [answered]
May I ask the contact email of the main person in charge or manager of nvm-window?
请问nvm-window主要负责人或主要管理者的联系邮箱是什么？

**Comment by coreybutler** [comment, marked as answer]
I don't post my email address publicly, but I have a contact form available at https://forms.gle/2kLRXZAe9KfTB7as9
